pkg/sse/stream.go

```diff
@@ -0,0 +1,138 @@
+// Copyright 2022 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sse
```

Comment:
Nit, this can be more generic like `streaming` or `stram`. I mean, the `sse` context is given when it is called to this package specifying the `sse` URL, but the code is completely generic and it is stream the logs from a URL. It could be sse or other thing.

Also, as we talked offline, destroy all generates logs with other format, so the piece of code parsing the body should be somehow specified from outside